README.md

```diff
@@ -0,0 +1,195 @@
+# Daily Art Prompt Service (Spring Boot)
+
+# Java
+
+- check version `java -version`
+
+## Hello World
+
+```java
+public class HelloWorld {
+  public static void main(String[] args) {
+    System.out.println("Hello World!");
+  }
+}
+```
+
+## IntelliJ
+   
+- To run java code you first need to compile it. To compile code in the terminal use command: `javac fileName.java`
+- If the file is successfully compiled it will create a fileName.class file that can be run
+- To run this file use the command: `java fileName`
+
+- You can also run the app with IntelliJ configurations. To build the gradle project make sure you right click on build.gradle and click 'import Gradle Project' after this you can then click the play button on the upper right to run the project
+
+## Comments
+
+- When comments are short we use the single-line syntax: `//`
+- When comments are long we use the multi-line syntax: `/* */`
```

Comment:
same here about being on the cheat sheet under a Java version. Then you can use this wiki as your "spring boot" example